### PR TITLE
Terraform python testing

### PR DESCRIPTION
**Pull request by kbalk**
LocalStack and `tftest` are used to perform integration testing.  New `Makefile` targets have been added to start/stop/cleanup LocalStack and to run the integration tests.

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.0.1
```
Comment (lorengordon) [inline CHANGELOG.md:7]: The version bump was for `0.1.0`, so I think this entry should match...

Comment (kbalk) [reply CHANGELOG.md:7]: Good catch ... I vacillated between whether is should be minor or patch and make different choices at different times.  Will fix.

File: CHANGELOG.md [hunk 2]
```diff
@@ -4,6 +4,16 @@ All notable changes to this project will be documented in this file.
 
 The format is based on [Keep a Changelog](http://keepachangelog.com/) and this project adheres to [Semantic Versioning](http://semver.org/).
 
+### 0.1.0
+
+**Commit Delta**: [Change from 0.0.0 release](https://github.com/plus3it/terraform-aws-org-new-account-iam-role/compare/0.0.0...0.0.1)
```
Comment (lorengordon) [inline CHANGELOG.md:9]: This link also references 0.0.1 instead of 0.1.0...

Comment (kbalk) [reply CHANGELOG.md]: Done

**Review by lorengordon — APPROVED** [review]
Let's do it! 🚀

**Pull request merged**